README.md

```diff
@@ -1 +1,4 @@
-# CapstoneProject
\ No newline at end of file
+# CapstoneProject
+
+# Map dependencies
+run npm install @angular/google-map to start running and using Google Maps Api on Angular
```

Comment:
This should just be part of the app dependencies by running "npm install @angular/google-maps --save". Then when someone downloads the project, they just need to run "npm install" and all dependencies will run. The problem with doing it this way (besides it just being more work to set up) is that the @angular/google-maps project is constantly being updated, and if a newer version is uploaded, it might not work the same way as the one you're developing with. Saving the dependency locks in a certain version.